python/cudf_kafka/cudf_kafka/_lib/kafka.pxd

```diff
@@ -11,14 +11,21 @@ from cudf._lib.cpp.io.types cimport datasource
 from cudf._lib.io.datasource cimport Datasource
 
 
+cdef extern from "kafka_callback.hpp" \
+        namespace "cudf::io::external::kafka" nogil:
+    ctypedef object (*kafka_oauth_callback_t)()
```

Comment:
Can we make this name consistent with the C++ name? The `*_t` suffixes are technically reserved by the POSIX standard. I don't know if we're consistent with that in general, but since this type alias already has a different name in C++ we may as well match that.